.github/workflows/build.yml

```diff
@@ -54,7 +55,7 @@ jobs:
       run: |
         echo "$HOME/.local/bin" >> "$GITHUB_PATH"
         echo "INSTALL_REF=${GITHUB_REF##*/}" >> "$GITHUB_ENV"
-        echo "INSTALL_NAME=lite-xl-${GITHUB_REF##*/}-linux-$(uname -m)" >> "$GITHUB_ENV"
+        echo "INSTALL_NAME=lite-xl-${GITHUB_REF##*/}-linux-$(uname -m)-portable" >> "$GITHUB_ENV"
```

Comment:
Because that is the file name generated when using the --portable flag which last time I tested was needed to properly get the lite-xl linux packaged .tar to actually work...

Reply:
Our current releases don't add `-portable` to the name, so I don't think we need to rename this.